.gitignore

```diff
@@ -21,8 +21,12 @@ fuzzing/**/findings/
 .*.swn
 .*.swo
 .*.swp
+*.save
+*.rej
 \#*\#
 cachegrind.out*
+# Core dumps
+core
```

Comment:
Given that we have a directory `core/` I don't know if this is a good idea to ignore...

Reply:
Changes in `core/` are still displayed with this. I think for this to capture the core directory we would have to add `core/` here.